fastlane/Fastfile

```diff
@@ -397,6 +403,24 @@ ENV["validate_translations"]="lintVanillaRelease"
     end
   end
 
+  #####################################################################################
+  # trigger_release_build
+  # -----------------------------------------------------------------------------------
+  # This lane triggers a stable release build on CI
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane trigger_release_build [branch_to_build:<branch_name>]
```

Comment:
small nit: param is not really optional so remove the `[]`

```suggestion
  # bundle exec fastlane trigger_release_build branch_to_build:<branch_name>
```

Reply:
Right! Good catch!